janus_server/src/aggregator.rs

```diff
@@ -159,7 +160,7 @@ where
             role,
             verify_param,
             report_recipient,
-            agg_auth_key,
+            agg_auth_keys,
```

Comment:
We should check that `agg_auth_keys` is non-empty up here (and return an error if not) because we use `aggregator.agg_auth_keys.first().unwrap()` later.